asset-transfer-basic/application-javascript/.gitignore

```diff
@@ -0,0 +1,80 @@
+#
```

Comment:
Worth sliming down from the default.. most is irrelevant

Reply:
done.